lib/mongodb/queries.nit

```diff
@@ -196,6 +230,77 @@ class MongoMatch
 		op("$nin", field, new JsonArray.from(values))
 		return self
 	end
+
+	# Logical `or`
+	#
+	# https://docs.mongodb.com/manual/reference/operator/query/or/#op._S_or
+	#
+	# The `$or` operator performs a logical OR operation on an array of two or
+	# more `expressions` and selects the documents that satisfy at least one of
+	# the `expressions`.
+	#
+	# The $or has the following syntax:
+	#
+	# ~~~json
+	# { field: { $or: [ { <expression1> }, { <expression2> }, ... , { <expressionN> } ] } }
+	# ~~~
+	fun lor(field: nullable String, expressions: Array[Jsonable]): MongoMatch do
+		op("$or", field, new JsonArray.from(expressions))
+		return self
+	end
+
+	# Logical `and`
+	#
+	# https://docs.mongodb.com/manual/reference/operator/query/and/#op._S_and
+	#
+	# The `$and` operator performs a logical AND operation on an array of two or
+	# more `expressions` and selects the documents that satisfy all of the `expressions`.
+	#
+	# The $and has the following syntax:
+	#
+	# ~~~json
+	# { field: { $and: [ { <expression1> }, { <expression2> }, ... , { <expressionN> } ] } }
+	# ~~~
+	fun land(field: nullable String, expressions: Array[Jsonable]): MongoMatch do
```

Comment:
try to compile the following code:

~~~nit
fun and do end
~~~

The `l` stands for `logical` since I needed a prefix.

Do you have a better suggestion?